build_tools/benchmarks/common/benchmark_thresholds.py

```diff
@@ -40,6 +40,11 @@ def get_threshold_str(self):
 # Order matters here: if multiple regexes match a single benchmark, the first
 # match is used.
 BENCHMARK_THRESHOLDS = [
+    # Compilation time measurement is very stable right now. Use a large
+    # threshold until we make it stable.
+    BenchmarkThreshold(re.compile(r"compilation-time"), 50,
```

Comment:
It seems like we should update the threshold on Line 87 `COMPILATION_TIME_THRESHOLDS = [...]` instead of here.

Reply:
Ha, okay it's split. Thanks for pointing out!